src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,39 +38,47 @@ public class RobotContainer {
   protected Limelight lime;
   protected LightSensor cubeSensor;
   protected LightSensor coneSensor;
-  private SendableChooser<PathPlannerPath> pathChooser;
+  private SendableChooser<Command> autonChooser;
+  private WaitCommand shooWaitCommand;
   protected IntakeSubsystem intake;
   protected ShooterSubsystem shooter;
   protected ArmSubsystem arm;
   protected CommandXboxController armController;
   protected CommandXboxController driveController;
-
+  private SendableChooser<PathPlannerPath> pathChooser;
   public RobotContainer(
-      ShooterSubsystem newShooter, IntakeSubsystem newIntake, ArmSubsystem newArm, Limelight lime, SwerveSubsystem drive) {
```

Comment:
Doesn't look like any of these are used in Robot.java. Can we just create them in RobotContainer.java instead of passing them in? Delete them out of Robot.java?